detect.go

```diff
@@ -22,15 +21,15 @@ type BuildPlanMetadata struct {
 // and requires cpython and pip at build.
 func Detect() packit.DetectFunc {
 	return func(context packit.DetectContext) (packit.DetectResult, error) {
-		_, err := os.Stat(filepath.Join(context.WorkingDir, "requirements.txt"))
+		exists, err := fs.Exists(filepath.Join(context.WorkingDir, "requirements.txt"))
 		if err != nil {
-			if errors.Is(err, os.ErrNotExist) {
-				return packit.DetectResult{}, packit.Fail
-			}
-
 			return packit.DetectResult{}, err
```

Comment:
@ForestEckhardt are you willing to add a test for this case?

Reply:
Not entirely sure I know what you mean? There is already test case for every outcome. I basically just rewrote this logic `fs` to make it easier to read.